gradio/themes/app.py

```diff
@@ -0,0 +1,139 @@
+
+    with gr.Row():
+        with gr.Column(variant="panel", scale=1):
+            gr.Markdown("## Panel 1")
+            radio = gr.Radio(
+                ["A", "B", "C"],
+                label="Radio",
+                info="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.",
+            )
+            drop = gr.Dropdown(["Option 1", "Option 2", "Option 3"], show_label=False)
+            drop_2 = gr.Dropdown(
+                ["Option A", "Option B", "Option C"],
+                multiselect=True,
+                value=["Option A"],
+                label="Dropdown",
+                interactive=True,
+            )
+            check = gr.Checkbox(label="Go")
+        with gr.Column(variant="panel", scale=2):
+            img = gr.Image(
+                "https://gradio.app/assets/img/header-image.jpg", label="Image"
+            ).style(height=320)
+            with gr.Row():
+                go_btn = gr.Button("Go", label="Primary Button", variant="primary")
+                clear_btn = gr.Button(
+                    "Clear", label="Secondary Button", variant="secondary"
+                )
+
+                def go(*args):
+                    time.sleep(3)
+                    return "https://gradio.app/assets/img/header-image.jpg"
+
+                go_btn.click(go, [radio, drop, drop_2, check, name], img, api_name="go")
+
+                def clear():
+                    time.sleep(0.2)
+                    return None
+
+                clear_btn.click(clear, None, img)
+
+            with gr.Row():
+                btn1 = gr.Button("Button 1").style(size="sm")
+                btn2 = gr.UploadButton().style(size="sm")
+                stop_btn = gr.Button("Stop", label="Stop Button", variant="stop").style(
+                    size="sm"
+                )
+
+    with gr.Row():
+        gr.Dataframe(value=[[1, 2, 3], [4, 5, 6], [7, 8, 9]], label="Dataframe")
+        gr.JSON(
+            value={"a": 1, "b": 2, "c": {"test": "a", "test2": [1, 2, 3]}}, label="JSON"
+        )
+        gr.Label(value={"cat": 0.7, "dog": 0.2, "fish": 0.1})
+        gr.File()
+    with gr.Row():
+        gr.ColorPicker()
+        gr.Video("https://gradio-static-files.s3.us-west-2.amazonaws.com/world.mp4")
+        gr.Gallery(
+            [
+                (
+                    "https://gradio-static-files.s3.us-west-2.amazonaws.com/lion.jpg",
+                    "lion",
+                ),
+                (
+                    "https://gradio-static-files.s3.us-west-2.amazonaws.com/logo.png",
+                    "logo",
+                ),
+                (
+                    "https://gradio-static-files.s3.us-west-2.amazonaws.com/tower.jpg",
+                    "tower",
+                ),
+            ]
+        )
```

Comment:
Currently, the Gallery is taking up too much vertical space
```suggestion
        ).style(height="200px", grid=2)
```